test/compositemark.test.ts

```diff
@@ -98,4 +98,349 @@ describe("normalizeErrorBar", () => {
         });
       }, Error, 'Neither x2 or y2 provided');
     });
+ });
+
+
+describe("normalizeBox", () => {
+  it("should produce an error if both axes are continuous", () => {
+    assert.throws(() => {
+      normalize({
+        "description": "A box plot showing median, min, and max in the US population distribution of age groups in 2000.",
+        "data": {"url": "data/population.json"},
+        mark: "box",
+        encoding: {
+          "x": {"field": "people","type": "quantitative"},
+          "y": {
+            "field": "people",
+            "type": "quantitative",
+            "axis": {"title": "population"}
+          },
+          "size": {"value": 5},
+          "color": {"value" : "white"}
+        }
+      });
+    }, Error, 'Need one continuous and one discrete axis');
+  });
+
+  it("should produce an error if both axes are discrete", () => {
+    assert.throws(() => {
+      normalize({
+        "description": "A box plot showing median, min, and max in the US population distribution of age groups in 2000.",
+        "data": {"url": "data/population.json"},
+        mark: "box",
+        encoding: {
+          "x": {"field": "age","type": "ordinal"},
+          "y": {
+            "field": "age",
+            "type": "ordinal",
+            "axis": {"title": "age"}
+          },
+          "size": {"value": 5},
+          "color": {"value" : "white"}
+        }
+      });
+    }, Error, 'Need one continuous and one discrete axis');
+  });
+
+  it("should produce correct layered specs for vertical boxplot with min and max lower and upper ticks", () => {
```

Comment:
That one has both x and y.  I'm looking for two more: one with only x, one with only y.